default.nix

```diff
@@ -227,89 +227,12 @@ rec {
     packages = [ ];
   };
 
-  demo-system =
-    module:
-    let
-      nixosSystem =
-        args:
-        import (sources.nixpkgs + "/nixos/lib/eval-config.nix") (
-          {
-            inherit lib;
-            system = null;
-          }
-          // args
-        );
-    in
-    nixosSystem {
-      system = "x86_64-linux";
-      modules = [
-        module
-        (sources.nixpkgs + "/nixos/modules/profiles/qemu-guest.nix")
-        (sources.nixpkgs + "/nixos/modules/virtualisation/qemu-vm.nix")
-        (
-          { config, ... }:
-          {
-            users.users.nixos = {
-              isNormalUser = true;
-              extraGroups = [ "wheel" ];
-              initialPassword = "nixos";
-            };
-
-            users.users.root = {
-              initialPassword = "root";
-            };
-
-            security.sudo.wheelNeedsPassword = false;
-
-            services.getty.autologinUser = "nixos";
-            services.getty.helpLine = ''
-
-              Welcome to NGIpkgs!
-            '';
-
-            services.openssh = {
-              enable = true;
-              settings = {
-                PasswordAuthentication = true;
-                PermitEmptyPasswords = "yes";
-                PermitRootLogin = "yes";
-              };
-            };
-
-            system.stateVersion = "25.05";
-
-            networking.firewall.enable = false;
-
-            virtualisation = {
-              memorySize = 4096;
-              cores = 4;
-              graphics = false;
-
-              qemu.options = [
-                "-cpu host"
-                "-enable-kvm"
-              ];
-
-              # ssh + open service ports
-              forwardPorts = map (port: {
-                from = "host";
-                guest.port = port;
-                host.port = port + 10000;
-                proto = "tcp";
-              }) config.networking.firewall.allowedTCPPorts;
-            };
-          }
-        )
-      ] ++ extendedNixosModules;
-    };
-
-  demo =
-    module:
-    pkgs.writeShellScript "demo-vm" ''
-      exec ${(demo-system module).config.system.build.vm}/bin/run-nixos-vm "$@"
-    '';
-
-  # $ nix-build . -A demo-test
-  # $ ./result
-  demo-test = demo ./projects/Cryptpad/demo.nix;
+  demo = import ./infra/demo {
```

Comment:
Hm, that's a tough one. I don't think this has anything to do with infra. How about putting it in overview? Maybe that directory is a misnomer (it's been called that way because it simply lists all applications), but it's our UI and that's where the demo will show up.

Another options would be the maintainers directory where all we currently store are some instructions for using the cache and remote builder. One could argue that running the local demo can also be part of the contributor workflow.